doc/io.rst

```diff
@@ -122,14 +124,15 @@ netCDF file. However, it's often cleaner to use a ``with`` statement:
     with xray.open_dataset('saved_on_disk.nc') as ds:
         print(ds.keys())
 
-.. note::
-
-    Although xray provides reasonable support for incremental reads of files on
+..  Although xray provides reasonable support for incremental reads of files on
     disk, it does not yet support incremental writes, which is important for
     dealing with datasets that do not fit into memory. This is a significant
     shortcoming that we hope to resolve (:issue:`199`) by adding the ability to
     create ``Dataset`` objects directly linked to a netCDF file on disk.
 
+Reading encoded data
+~~~~~~~~~~~~~~~~~~~~
+
 NetCDF files follow some conventions for encoding datetime arrays (as numbers
 with a "units" attribute) and for packing and unpacking data (as
 described by the "scale_factor" and "_FillValue" attributes). If the argument
```

Comment:
The reader might think that "_FillValue" is part of the packing/unpacking descriptor and not  the missing data.  I think you meant "scale_factor" and "add_offset".